mart/attack/composer.py

```diff
@@ -15,28 +15,13 @@
 class Composer(abc.ABC):
     def __call__(
         self,
-        perturbation: torch.Tensor | Iterable[torch.Tensor],
+        perturbation: torch.Tensor,
         *,
-        input: torch.Tensor | Iterable[torch.Tensor],
-        target: torch.Tensor | Iterable[torch.Tensor] | Iterable[dict[str, Any]],
+        input: torch.Tensor,
+        target: torch.Tensor | dict[str, Any],
```

Comment:
These changes make me think this change is a regression? Why do we not accept `Iterable[torch.Tensor]` anymore?

Reply:
OK. I reverted changes to `Composer` and created a new `Modality(Composer)`.